TradeItIosTicketSDK2/TradeItYahooLauncher.swift

```diff
@@ -96,7 +96,7 @@ import SafariServices
         }
     }
 
-    public func launchTrading(
+    @objc public func launchTrading(
         fromViewController viewController: UIViewController,
         withOrder order: TradeItOrder,
         onViewPortfolioTappedHandler: @escaping OnViewPortfolioTappedHandler
```

Comment:
Verify this hook still works from obj-c as we use this in the Finance app.
Perhaps consider adding an invocation of this to the obj-c example?

The `typealias` for the handler can't be annotated with `obj-c` but passing in the long-form signature should work.